composer.json

```
@@ -0,0 +1,14 @@
+{
+    "name": "netzstrategen/woocommerce-permalink-structure",
+    "type": "wordpress-plugin",
+    "authors": [
+        {
+            "name": "netzstrategen",
+            "email": "hallo@netzstrategen.com",
+            "homepage": "https://netzstrategen.com"
+        }
+    ],
+    "require": {
+        "php": ">=7.0.0"
```

I think we're actually requiring at least 7.1 in all of our code, which is a big difference due to the null coalesque operator.
```suggestion
        "php": ">=7.1.0"
```

---

Thanks for your help!